src/client/Microsoft.Identity.Client/Internal/Requests/Interactive/AuthCodeRequestComponent.cs

```diff
@@ -176,7 +176,11 @@ public Uri GetAuthorizationUriWithoutPkce()
                 authorizationRequestParameters[kvp.Key] = kvp.Value;
             }
 
-            if (_interactiveParameters.Prompt.PromptValue != Prompt.NoPrompt.PromptValue)
+            if (_interactiveParameters.Prompt == Prompt.NotSpecified)
```

Comment:
why have the NotSpecified if it's using selectAccount? i'd think if developers did not set a prompt we shouldn't send any prompt value (that's actually what b2c would prefer at least, so they (ests) can determine which sign-in screen to show based on the current session cookie). I don't see the value in adding a new prompt value.